components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt

```diff
@@ -39,6 +39,11 @@ interface Toolbar {
      */
     var private: Boolean
 
+    /**
+     * Sets/gets indicates if this toolbar belongs to an external session e.g Custom tab or PWA.
+     */
+    var isExternal: Boolean
+
```

Comment:
As mentioned above: I would only add a separator flag to `browser-toolbar` and not add this to the concept.

Reply:
Updated ✅